internal/lobbyServer/lobby.go

```diff
@@ -135,6 +139,30 @@ func (s *LobbyServer) updatePlayers(g *gameserver.GameServer) {
 	}
 }
 
+func (s *LobbyServer) updateRoom(g *gameserver.GameServer, name string) {
+	if g == nil {
```

Comment:
I don't think this is ever possible, since the function that calls it also checks if g is nil

Reply:
That is true, I'll remove it.